all/native/routing/OSRMRoutingProxy.h

```diff
@@ -7,7 +7,7 @@
 #ifndef _CARTO_OSRMROUTINGPROXY_H_
 #define _CARTO_OSRMROUTINGPROXY_H_
 
-#ifdef _CARTO_ROUTING_SUPPORT
+//#ifdef _CARTO_ROUTING_SUPPORT
```

Comment:
I do not see why this should be for commented out

Reply:
thanks fixed!